test/controllers/NirmsNumberControllerSpec.scala

```diff
@@ -376,66 +379,7 @@ class NirmsNumberControllerSpec extends SpecBase with MockitoSugar {
         }
       }
 
-      "must redirect to Profile for a POST and submit data if value is different from original" in {
```

Comment:
Tests not relevant anymore are removed, there is one that was still relevant of this type.